test/electionguard/test_chaum_pedersen.cpp

```diff
@@ -195,7 +220,85 @@ TEST_CASE("Disjunctive CP Proof encryption of one with precomputed values invali
     PrecomputeBufferContext::clear();
 }
 
-TEST_CASE("Constant CP Proof encryption of zero" * doctest::may_fail())
+TEST_CASE("Ranged CP Proof encryption of zero generates valid proof")
```

Comment:
TODO: all 3 of the following tests differ only on the selected value and can be combined

Reply:
fixed in my latest commit